connect-button/src/main/java/com/ifttt/connect/ui/BaseConnectButton.java

```diff
@@ -459,13 +473,60 @@ public void onLayoutChange(View v, int left, int top, int right, int bottom, int
                 // Cancel potential disable connection API call.
                 buttonApiHelper.cancelDisconnect();
 
-                if (buttonApiHelper.shouldPresentEmail(getContext())) {
-                    buildEmailTransitionAnimator(0).start();
+                if (connection.status == never_enabled || connection.status == unknown) {
```

Comment:
is `unknown` status a real  possibility?

Reply:
Yes, I think it is documented in the Connect API where the status could be null.